homeassistant/components/vera/sensor.py

```diff
@@ -101,8 +104,7 @@ def update(self) -> None:
                 self.current_value = value
             self.last_changed_time = time
         elif self.vera_device.category == veraApi.CATEGORY_POWER_METER:
-            power = convert(self.vera_device.power, float, 0)
-            self.current_value = int(round(power, 0))
+            self.current_value = self.vera_device.power
```

Comment:
This is a code change, but the underlying library is typed and returns power as an int